SkillFeatureProjectMars/StepDefinitions/DeleteSkillStepDefinitions.cs

```diff
@@ -0,0 +1,40 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using SkillFeature_Mars.Drivers;
+using SkillFeature_Mars.Pages;
+using System;
+using TechTalk.SpecFlow;
+
+namespace SkillFeature_Mars.StepDefinitions
+{
+    [Binding]
+    public class DeleteSkillStepDefinitions : CommonDriver
+    {
+        DeleteSkillPage deletePageObj = new DeleteSkillPage();
```

Comment:
make use of constructors
Please see
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors